### ISSUE TITLE
Add ATLAS

### ISSUE THREAD

**Issue by danholdaway** [JCSDA/jedi-stack#50]
As mentioned on the call yesterday it might be good to add scripts for installing ATLAS. The ctests take an eternity.

**Comment by mmiesch** [comment]
Yep - good idea - I can take this on.  I was finding that it wasn't too bad on my laptop with gnu/openmpi but yes, if you had to do all those tests every time it would get annoying.

**Comment by danholdaway** [comment]
Maybe Discover was running slow when I ran the tests. How long did they take for you?

**Comment by mmiesch** [comment]
I just ran it again - only 26 seconds.  Several test failures, though.

~~~~~
Label Time Summary:
atlas         =  25.91 sec*proc (105 tests)
executable    =  89.15 sec*proc (185 tests)
fckit         =   0.90 sec*proc (13 tests)
fortran       =   6.30 sec*proc (34 tests)
fv3jedi       = 755.78 sec*proc (61 tests)
ioda          =   9.69 sec*proc (11 tests)
mpi           = 986.59 sec*proc (108 tests)
oops          = 152.18 sec*proc (246 tests)
openmp        = 217.51 sec*proc (80 tests)
saber         = 106.04 sec*proc (75 tests)
script        = 1154.51 sec*proc (404 tests)
ufo           = 193.17 sec*proc (78 tests)

Total Test time (real) = 1244.58 sec

The following tests FAILED:
	 27 - atlas_test_metadata (Failed)
	 28 - atlas_test_trace (Failed)
	 30 - atlas_test_haloexchange (Failed)
	 31 - atlas_test_gather (Failed)
	 86 - atlas_test_parfields (Failed)
	 87 - atlas_test_halo (Failed)
	 88 - atlas_test_distmesh (Failed)
	 90 - atlas_test_mesh_node2cell (Failed)
	 97 - atlas_fctest_functionspace (Failed)
	 99 - atlas_test_structuredcolumns (Failed)
	103 - atlas_test_stencil_parallel_mpi4 (Failed)
	104 - atlas_test_stencil_parallel_mpi16 (Failed)
	308 - test_l95_letkf_compare (Failed)
	436 - test_qg_4dvar_saddlepoint_compare (Failed)
	439 - test_qg_letkf_compare (Failed)
	548 - test_fv3jedi_hofx_gfs_compare (Failed)
~~~~~

**Comment by danholdaway** [comment]
OK, maybe not an issue then. When I ran yesterday I was seeing some tests that took three minutes.

**Comment by mmiesch** [comment]
I'm working on a jedi-stack branch now to add this.  It seems to work but one caveat is that atlas requires fckit.  So, if we take atlas out of the bundle and load it as part of the stack, I think we'll have to do the same for fckit.

**Comment by mmiesch** [comment]
I'm moving this to the icebox.  There is a branch of jedi-stack that has an atlas install script but it's not working.  ecbuild is unable to find it, even when I carefully set the ${ATLAS_PATH} variable to the install directory and the ${atlas_DIR} variable to the ${ATLAS_PATH}/share/atlas/cmake directory where the config files are.  I tried various permutations of the upper/lower case on atlas and I tried brining ATLAS_ROOT into the mix as well but nothing seemed to work. 

However, as we discussed in our core team meeting this morning, we'll want to build atlas as part of the bundles at least initially because we are likely to change it.  By the time it's stable enough to install as part of the stack, we'll likely be using a newer version of ecbuild and this issue may just go away.
